301-multi-tier-loadbalancing/azuredeploy.parameters.json

```diff
@@ -0,0 +1,47 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    
+    "Region1-location-name": {
+      "value": "Central US"
+    },
+    "Region2-location-name": {
+      "value": "North Europe"
+    },
+    "trafficManagerProfile_name": {
+      "value": "tmcptest"
+    },
+    "adminuser": {
+      "value": "admin123"
+    },
+    "adminuserPassword": {
+      "value": "TestUser@123"
+    },
+
+    "App-prefix": {
+      "value": "cp"
+    },
+    "loadbalancer-dns-prefix": {
+      "value": "lb"
```

Comment:
should probably use GEN-UNIQUE: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE